docs/site/src/demos/drawers/DockedDrawer.js

```diff
@@ -0,0 +1,108 @@
+// @flow weak
+
+import React, { Component } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import Drawer from 'material-ui/Drawer';
+import Button from 'material-ui/Button';
+import {
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+} from 'material-ui/List';
+import Divider from 'material-ui/Divider';
+import InboxIcon from 'material-ui/svg-icons/inbox';
+import DraftsIcon from 'material-ui/svg-icons/drafts';
+import StarIcon from 'material-ui/svg-icons/star';
+import SendIcon from 'material-ui/svg-icons/send';
+import MailIcon from 'material-ui/svg-icons/mail';
+import DeleteIcon from 'material-ui/svg-icons/delete';
+import ReportIcon from 'material-ui/svg-icons/report';
+
+
+const styleSheet = createStyleSheet('DockedDrawer', () => ({
+  list: {
+    width: 250,
+    flex: 'initial',
+  },
+  remainder: {
+    flex: 1,
+  },
+}));
+
+export default class DockedDrawer extends Component {
+  state = {
+    open: false,
+  };
+
+  handleOpen = () => this.setState({ open: true });
+  handleClose = () => this.setState({ open: false });
+
+  render() {
+    const classes = this.context.styleManager.render(styleSheet);
+    return (
+      <div>
+        <Button onClick={this.handleOpen}>Open Drawer</Button>
+        <Drawer
+          open={this.state.open}
+          onRequestClose={this.handleClose}
+          docked
+        >
+          <List className={classes.list} padding={false} onClick={this.handleClose}>
```

Comment:
I'm wondering. What about sharing this List component between the two demos. I don't think that it's so important for the examples. We would get
- A shorter/clearer demo
- DRY

We only lose a bit of indirection.